app/src/main/res/mipmap-anydpi-v26/ic_app_launcher.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/3c8383d74d299a849c1a157003dd3fd1ed3fb9c9/app/src/main/res/mipmap-anydpi-v26/ic_app_launcher.xml#L2"></span>The application adaptive icon is missing a monochrome tag</td>
    </tr>
  </tbody>
</table>